src/SDKs/CognitiveServices/dataPlane/Search/BingCustomSearch/BingCustomSearch/Microsoft.Azure.CognitiveServices.Search.CustomSearch.csproj

```diff
@@ -6,10 +6,10 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.CognitiveServices.Search.CustomSearch</PackageId>
     <Description>This client library provides access to the Microsoft Cognitive Services Custom Search API.</Description>
-    <VersionPrefix>1.2.0-preview</VersionPrefix>
```

Comment:
The stable version should be at least equivalent to the preview version and lower than the major version
You should ideally have the stable version as `1.2.0` here

Reply:
Made all stable versions as 1.2.0